### PR TITLE
TEP006: configuration system improvement

### PR DESCRIPTION
**Pull request by wkerzendorf**
This suggests a change to the configuration system that will enable us to easier add options and make the code more maintainable and less error prone.

### CODE REVIEW

File: TEP006_configuration_tags.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+shells and abundances that are uniform throughout the model using one data
+scheme. But also provide files that described a complex models with unevenly
+spaced shells and different abundances for the same model. Depending on the
+model, TARDIS needs to parse the information in different ways to convert
+them to the common format needed within the code (DataFrames).
+
+The current way the code deals with the input of configuration ways is divided
+into several steps:
+
+1. Parse the YAML file with PyYaml
```
Comment (unoebauer) [inline TEP006_configuration_tags.rst:37]: Formatting is messed up here - it's indented in the web rendering

**Comment by unoebauer** [comment]
Looks reasonable - let's adopt it.

File: TEP006_configuration_tags.rst (hunk 1 above)
Comment (wkerzendorf) [inline TEP006_configuration_tags.rst:37]: not sure how to fix this. can we leave it for now @unoebauer

**Comment by wkerzendorf** [comment]
@unoebauer merge at will

**Pull request merged**